java-client/session/src/main/java/io/deephaven/client/impl/ExportStates.java

```diff
@@ -224,7 +261,7 @@ class State {
         State(TableSpec table, int exportId) {
             this.table = Objects.requireNonNull(table);
             this.exportId = exportId;
-            this.children = new LinkedHashSet<>();
+            this.children = new CopyOnWriteArraySet<>();
```

Comment:
I understand we want this to avoid CME's on remove during an iteration. Can we build a list to remove and do a removeAll instead? I'm worried about adding COW data structures with user-facing adds.

Reply:
I more thoroughly traced exactly the part that was causing concurrent mod, and it was actually the mitigation; now that the Batch / ETCR race has been fixed, I can get rid of the mitigation and revert this change.